student.rb

```diff
@@ -1,12 +1,20 @@
 require './person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(classroom, age, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
+    add_classroom(classroom)
   end
 
   def play_hooky
     '¯(ツ)/¯'
   end
+
+  def add_classroom(classroom)
+    @classroom = classroom
+    classroom.students << self
```

Comment:
- [ ] make sure that the student is only pushed to the classroom.students unless the classroom.students includes the student already to avoid duplicates.

Reply:
Got it, @mikethreels. 👍